### PR TITLE
Revamp of 'Refunds' section, most notably of email notifications section

### PR DESCRIPTION
**Pull request by bahmady**
### Context
Language/style (relatively minor) revamp of the 'Refunds' section, including rework notably of the content surrounding email notifications

### Guidance to review
I've added my own review comment to this for potential reviewers: would be great to get this sorted before proceeding

### CODE REVIEW

File: source/refunding_payments/index.html.md.erb [hunk 1]
```diff
@@ -79,54 +78,54 @@ with a previous refund of £30:
...
+two refunds of £2 each.
+
+As another example, you may have specified the `refund_amount_available` as
+£5. The first refund request for £2 still succeeds, leaving £3 available for
+refund. When the accidental second request arrives, the payment has a
+``refund_amount_available`` of £5 even though only £3 is available. GOV.UK
+Pay determines that it’s a stale request, and rejects the refund request.
+
+Your service should track the expected refund amount available. It should
+return a `refund_amount_available` value whenever you request a refund via
```
Comment (bahmady) [inline source/refunding_payments/index.html.md.erb]: Not sure if I'm going mad but I'm confused about whether this particular line should reference `refund_amount_available` or `amount_available`

Comment (katiebates) [reply source/refunding_payments/index.html.md.erb]: refund_amount_available forms part of the request body when a refund is submitted via API

File: source/refunding_payments/index.html.md.erb [hunk 2]
```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
...
+
+You can use the GOV.UK Pay admin tool to [send email notifications to
+users](https://selfservice.payments.service.gov.uk/email-notifications).
 
-End users are automatically notified by email about successful payments.  This
-depends on settings you have entered in the self-service admin site. Users are
-not notified when a payment has either failed or is refunded - either manually
-or via the API. You should arrange to notify end users about payment failures
-and refunds as appropriate for your service.
+Users are not notified when a payment has either failed or is refunded. You
```
Comment (katiebates) [inline source/refunding_payments/index.html.md.erb]: we do send refund email notifications now. these are enabled by default on new accounts, and can be switched off in the same place as payment confirmation emails.

File: source/refunding_payments/index.html.md.erb [hunk 3]
```diff
@@ -164,12 +163,11 @@ refund attempt that fails like this with an error code is not assigned a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/payment-id/find-payment-refund-by-id"
 target="blank">Find payment refund by ID</a> (link opens in new window)).
 
-If accepted by GOV.UK Pay, a refund may still go on to fail at the PSP. This
-may happen if the card involved is cancelled or has expired, or if your
-account with the PSP does not have enough funds to cover the refund.
+If a refund is accepted by GOV.UK Pay, it may still go on to fail at the PSP.
+This may happen if the card involved is cancelled or has expired, or if your
```
Comment (katiebates) [inline source/refunding_payments/index.html.md.erb]: In the 'refund destination' section - "If the original card expired or was canceled, the bank will attempt to credit the customer’s new card with the refund" - perhaps we could clarify this to make it clear that a cancelled or expired card isn't the sole reason why the refund attempt failed? Is there room for confusion here?

File: source/refunding_payments/index.html.md.erb [hunk 4]
```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
...
 
 Refunds are sent back to the source account of the original payment. It is not
 possible to refund a payment to another card or bank account. If the original
-card expired or was canceled, the bank will attempt to credit the customer’s
+card expired or was cancelled, the bank will attempt to credit the customer’s
 new card with the refund. If this attempt fails, or the customer does not have
-a new card, the refund will have to be made using another channel (eg bank
-transfer or cheque).
+a new card, the refund will have to be made using another method. For example,
+bank transfer or cheque.
```
Comment (katiebates) [inline source/refunding_payments/index.html.md.erb]: I think we should make it clearer here that it's the service's responsibility to use a different method to refund - GOV.UK Pay doesn't do it, nor does the bank.

File: source/refunding_payments/index.html.md.erb [hunk 5]
```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
...
 
 In the event of an error, GOV.UK Pay will not currently provide any more
 information. Please [contact
-us](/support_contact_and_more_information/#contact-us) if more information is
-required about why a refund failed.
+us](/support_contact_and_more_information/#contact-us) if you need more
+information required about why a refund failed.
 
-## Refunding from the GOV.UK Pay admin site
+## Refunding from the GOV.UK Pay admin tool
```
Comment (katiebates) [inline source/refunding_payments/index.html.md.erb]: can i doublecheck whether 'admin tool' is the preferred moniker throughout the rest of docs?

Comment (bahmady) [reply source/refunding_payments/index.html.md.erb]: It should be; I've had to try to clean up a lot of things like this as I've been going along but it's one of the things I've made a note to go back and check throughout the whole documentation before I finish on Pay

File: source/refunding_payments/index.html.md.erb (hunk 1 above)
Comment (bahmady) [reply source/refunding_payments/index.html.md.erb]: In which case presumably this should be `amount_available`?

**Comment by bahmady** [comment]
@katiebates see latest update

File: source/refunding_payments/index.html.md.erb (hunk 1 above)
Comment (katiebates) [reply source/refunding_payments/index.html.md.erb]: looks good

**Pull request merged**